thor/thor.php

```diff
@@ -993,12 +994,12 @@ function _build_db_structure()
 				$db_structure[$node->tagAttrs['id']]['type'] = 'text';
 			}
 			elseif (($node->tagName == 'radiogroup') || ($node->tagName == 'optiongroup')) {
-				$db_structure[$node->tagAttrs['id']]['type'] = 'tinytext';
+				$db_structure[$node->tagAttrs['id']]['type'] = 'text';
 			}
 			elseif ($node->tagName == 'checkboxgroup') {
 				$node_children = $node->tagChildren;
 				foreach ($node_children as $node2) {
-					$db_structure[$node2->tagAttrs['id']]['type'] = 'tinytext';
+					$db_structure[$node2->tagAttrs['id']]['type'] = 'text';
```

Comment:
How will this work for existing tables? Won't we need an upgrade script to convert these `tinytext` columns to `text` for existing forms?

Reply:
For expediency, I was planning to manually update the specific problem areas that I have in the IRB updates I'm making. My (intentionally-naive) assumption is that this is not a bug-producing problem elsewhere, or it would have come up sooner. 

I have to do other manual changes to the IRB forms' tables to accommodate field changes and rearrangements, so I intend to wrap all of that together.

If that is too lazy (it's too lazy) I can write an upgrade script, but I am certainly aiming for the shortcut here.

Reply:
That sounds OK to me. We will just need to file this detail away in case it rears its head later. I don't know of an easy way to address this globally.

Reply:
Changing existing tables could have surprising consequences. There could be two different strings stored for one logical value - truncated strings before the change, untruncated after.

David, do you have a sense of how many labels are long enough to be truncated now? It might not be that many -- in which case we can patch things up manually.

Reply:
I haven't done any kind of broad search. In the IRB, the main culprits are in the Summary Form, which has four different questions with fairly long labels on the radio options, each with four or five options in each group. 

The existing IRB Summary form predates the change from ENUM to (TINY)TEXT change for these fields, so this is a new issue. I still have to figure out how I will reconcile the existing data with the new form structure - I will probably convert these from ENUMs to TEXT.

Reply:
Last call on this. I think I'm clear to make this merge after a quick look again. Barring red flags, I'll make the merge tomorrow (12/22/2017)